pkg/controllers/disruption/controller.go

```diff
@@ -184,7 +185,8 @@ func (c *Controller) executeCommand(ctx context.Context, m Method, cmd Command)
 		methodLabel:            m.Type(),
 		consolidationTypeLabel: m.ConsolidationType(),
 	}).Inc()
-	logging.FromContext(ctx).Infof("disrupting via %s %s", m.Type(), cmd)
+	uid := uuid.NewUUID()
```

Comment:
Did we have a chance to look at controller-runtime to see if we could leverage their reconcile id rather than having to create our own concept?

Reply:
synced offline, going to punt this for a future refactor. At first look, I couldn't find an easy reference to the controller-runtime reconcile id, but for within disruption/orchestration, we're thinking it makes less sense for the Singleton controller abstraction.